views/templates/admin/tabs/display/checkout.tpl

```diff
@@ -0,0 +1,53 @@
+{*
+* 2007-2019 PrestaShop
+*
+* NOTICE OF LICENSE
+*
+* This source file is subject to the Academic Free License (AFL 3.0)
+* that is bundled with this package in the file LICENSE.txt.
+* It is also available through the world-wide-web at this URL:
+* http://opensource.org/licenses/afl-3.0.php
+* If you did not receive a copy of the license and are unable to
+* obtain it through the world-wide-web, please send an email
+* to license@prestashop.com so we can send you a copy immediately.
+*
+* DISCLAIMER
+*
+* Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+* versions in the future. If you wish to customize PrestaShop for your
+* needs please refer to http://www.prestashop.com for more information.
+*
+*  @author    PrestaShop SA <contact@prestashop.com>
+*  @copyright 2007-2019 PrestaShop SA
+*  @license   http://opensource.org/licenses/afl-3.0.php  Academic Free License (AFL 3.0)
+*  International Registered Trademark & Property of PrestaShop SA
+*}
+
+<div class="panel panel-default col-lg-8 col-lg-offset-1 col-md-8 col-md-offset-0" >
+    <div class="panel-heading">
+        {l s='Specific position for check out page' mod='psreassurance'}
+    </div>
+
+    <div class="col-xs-12 col-sm-12 col-md-12 col-lg-12 position-hook">
+        <label class="psr-float">
+            <div class="help-block customradiodesign">
+                <input type="radio" class="input_img js-show-all" name="PSR_HOOK_CHECKOUT" value="1" 
+                {if $psr_hook_checkout eq 1}checked="checked"{/if}
+                />
+                 <label for="PSR_HOOK_CHECKOUT"><span><span></span></span>{l s='Main column' mod='psagechecker'}</label><br /><br />
+                <img src="{$img_path}displayHome_active.jpg" width="150" height="150" class="psr-checkout-color{if $psr_hook_checkout eq 1} active{/if}">
```

Comment:
I'm not a HTML expert but I think some `alt` would be nice for users

Reply:
We are in the back office here

Reply:
Well ... still 😛 ? `alt` are useful for vision impaired people and people with a bad network, even in the BO ?